packages/nx-plugin/src/executors/e2e/e2e.impl.ts

```diff
@@ -59,7 +59,7 @@ async function runTests(
   context: ExecutorContext
 ): Promise<boolean> {
   const { success } = await jestExecutor(
-    { ...jestOptions, watch: false },
+    { runInBand: true, ...jestOptions, watch: false },
```

Comment:
unsure if we always want to do this?
it would make more sense to have this defaulted to `true` and then allow it to be changed if so desired.

Reply:
I think @AgentEnder and I still want to try to get rid of this executor all together and see if it can be configured purely via jest setup. TBD.

Reply:
ok. remove code here and set runInBand to true in schema.json